contracts/lbp/LBPManager.sol

```diff
@@ -51,25 +55,35 @@ contract LBPManager {
     }
 
     /**
-     * @dev                             initialize lbp wrapper contract
-     * @param _LBPFactory               LBP factory address
-     * @param _beneficiary              address of the receiver of the _primeDaoFeePercentage
-     * @param _name                     LBP name
-     * @param _symbol                   LBP symbol
-     * @param _tokens                   array of tokens sorted for the LBP
-     * @param _amounts                  array of amounts of tokens that would be added as liquidity.
-     * @param _weights                  array of start weights for respective tokens
-     * @param _startTimeEndTime         array of start time and end time
-     * @param _endWeights               array of end weights for respective tokens
-     * @param _swapFeePercentage        fee percentage for swapping
-     * @param _primeDaoFeePercentage    fee percentage for providing the LBP service
+     * @dev                             Initialize LBPManager.
+     * @param _LBPFactory               LBP factory address.
+     * @param _beneficiary              The address that receives the _primeDaoFeePercentage.
+     * @param _name                     Name of the LBP.
+     * @param _symbol                   Symbol of the LBP.
+     * @param _tokenList                Sorted array containing two parameters:
```

Comment:
numerically sorted ascending is correct, right?

```suggestion
     * @param _tokenList                Numerically sorted array (ascending) containing two addresses:
```

Reply:
Yes, and I'll add you're suggestion